src/sweet-module.js

```diff
@@ -101,17 +111,59 @@ export default class SweetModule {
         moreDirectives = false;
       }
 
-      if (S.isImportDeclaration(item)) {
-        imports.push((item: any));
-      } else if (S.isExportDeclaration(item)) {
-        exports.push((item: any));
-        body.push(item);
-        this.exportedNames = this.exportedNames.concat(extractNames(item));
-        if (S.isExportDefault(item)) {
-          this.defaultExport = Syntax.fromIdentifier('_default');
-          this.exportedNames = this.exportedNames.push(
-            ExpSpec(this.defaultExport)
+      if (item instanceof T.ImportDeclaration) {
+        imports.push(item);
+      } else if (item instanceof T.ExportDeclaration) {
+        if (S.isExport(item)) {
+          let decl = extractDeclaration(item);
+          let stmt = wrapStatement(decl);
+          let names = extractDeclarationNames(decl);
+          body.push(stmt);
+          let exp = new T.ExportFrom({
```

Comment:
That was added in shift spec es2016, we're on es6.

Definitely need to update but not in this PR.